.github/workflows/deploy.yml

```
@@ -3,45 +3,113 @@ on:
     runs-on: ubuntu-latest
     environment:
       name: conoha
     steps:
-      - uses: actions/checkout@v3
-      - uses: actions/setup-go@v3
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      # Rustバックエンドのビルド
+      - name: Setup Rust
+        uses: actions-rs/toolchain@v1
```

actionlint (1.7.4)

the runner of "actions-rs/toolchain@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)